docs/dev/ArtidfactsPublishing.md

```diff
@@ -1,22 +1,8 @@
-# How to upgrade Guava in Drill
```

Comment:
Please remove whole the file, since except for shaded guava, there is no need anymore for publishing libraries manual.

Reply:
I thought to it too. But this info can be useful for any other library shade, can't it be?